exo-setup/features/setup.feature

```diff
@@ -39,3 +39,8 @@ Feature: Setup of Exosphere applications
     And ExoCom uses this routing:
       | ROLE             | SENDS                       | RECEIVES                    | NAMESPACE |
       | external-service | users.list, users.create    | users.listed, users.created |           |
+
+  Scenario: set up the application with silenced services and dependencies
```

Comment:
Didn't realize that output was being silenced for `exo-setup` as well. I think that can be a potential source of confusion for the user. For now we should always log output when setting up the application and only allow the option to hide output during `exo-run`.

But it's good to know that if we do want this feature in the future it's really easy to implement.